application/src/main/java/org/thingsboard/server/service/resource/DefaultTbResourceService.java

```diff
@@ -76,13 +77,13 @@ public TbResourceInfo findResourceInfoById(TenantId tenantId, TbResourceId resou
     }
 
     @Override
-    public PageData<TbResourceInfo> findAllTenantResourcesByTenantId(TenantId tenantId, PageLink pageLink) {
-        return resourceService.findAllTenantResourcesByTenantId(tenantId, pageLink);
+    public PageData<TbResourceInfo> findAllTenantResourcesByTenantId(TbResourceInfoFilter tbResourceInfoFilter, PageLink pageLink) {
```

Comment:
it is better to use simple variable name that is more readable (just ```filter``` instead of ```tbResourceInfoFilter```)